pyoffers/models/application.py

```diff
@@ -0,0 +1,24 @@
+from pyoffers.utils import Filter
+
+from .core import Model, ModelManager
+
+
+class Application(Model):
```

Comment:
Is it required?

Reply:
Yes. Otherwise how would I add a manager?